src/test/java/com/cisco/cta/taxii/adapter/TcpServer.java

```diff
@@ -27,7 +27,7 @@
 
 class TcpServer extends Thread implements AutoCloseable {
 
-    private static int port = 9000;
+    private static int port = 19000;
```

Comment:
Why changing the port?

Reply:
Beacuse I had conflict with my localhost Sonar server and suppose testing server could have port > 10000 not to interfere with common services.